manifests/params.pp

```diff
@@ -5,7 +5,11 @@
 
   case $::osfamily {
     'RedHat': {
-      $plugin_prefix = 'rubygem-smart_proxy_'
+      if versioncmp($::operatingsystemmajrelease, '7') <= 0 {
```

Comment:
For future reference: we should start to use `$facts` to clearly mark it as a fact and also use modern (non-legacy) facts. However, this entire still needs to be converted, it's not that important. Just putting it here to as knowledge sharing.
```suggestion
      if versioncmp($facts['operatingsystemmajrelease'], '7') <= 0 {
```

Or the modern fact:
```suggestion
      if versioncmp($facts['os']['release']['major'], '7') <= 0 {
```